README.rst

```diff
@@ -17,6 +19,32 @@ Build and tests status
 |   :alt: Linux Master branch tests status                                           |
 +------------------------------------------------------------------------------------+
 
+
+Usage
+=====
+
+::
+
+    >>> from packageurl import PackageURL
+    >>> purl = PackageURL.from_string("pkg:maven/org.apache.commons/io@1.3.4")
+    >>> print(purl.to_dict())
+    {'type': 'maven', 'namespace': 'org.apache.commons', 'name': 'io', 'version': '1.3.4', 'qualifiers': None, 'subpath': None}
+    >>> print(purl.to_string())
+    pkg:maven/org.apache.commons/io@1.3.4
+    >>> print(str(purl))
+    pkg:maven/org.apache.commons/io@1.3.4
+    >>> print(repr(purl))
+    PackageURL(type='maven', namespace='org.apache.commons', name='io', version='1.3.4', qualifiers={}, subpath=None)
+
+
+Other utilities:
+
+- packageurl.contrib.django_models.PackageURLMixin is a Django abstract model mixin to use Package URLs in Django.
+- packageurl.contrib.purl2url.purl2url(purl) (and get_url) returns the download URL inferred from a Package URL.
```

Comment:
This API feels a bit awkward, we should only promote 1 way of calling the code: `purl2url.get_url(purl)` and `purl2url.get_purl(url)` in the documentation.

Reply:
Fixed. Thanks!